content/docs/central-dash/_index.md

```diff
@@ -0,0 +1,11 @@
++++
```

Comment:
Let's put all the central dashboard docs into one place, in the **components** section of the docs. So:

Move this page from `content/docs/central-dash/_index.md` to `content/docs/components/central-dash/_index.md`.

Reply:
Done!